Recrutify/Recrutify.Services/Services/SendQueueEmailService.cs

```diff
@@ -8,18 +13,31 @@ public class SendQueueEmailService : ISendQueueEmailService
         private readonly IFormEmailService _formEmailService;
         private readonly ISendEmailService _sendEmailService;
 
-        public SendQueueEmailService(IFormEmailService formEmailService, ISendEmailService sendEmailService)
+        public SendQueueEmailService(ISendEmailService sendEmailService, IFormEmailService formEmailService)
         {
             _formEmailService = formEmailService;
             _sendEmailService = sendEmailService;
         }
 
-        public void SendEmail()
+        public async Task SendEmail(List<CandidateDTO> candidates, StatusDTO status, ProjectDTO project)
         {
-            var requests = _formEmailService.GetEmailRequests();
+            IEnumerable<EmailRequest> requests;
+            switch (status)
+            {
+                case StatusDTO.Accepted:
+                    requests = _formEmailService.FormEmail(candidates, project, Constants.TemplatePath.AcceptanceTemplate);
+                    break;
+                case StatusDTO.Declined:
+                    requests = _formEmailService.FormEmail(candidates, project, Constants.TemplatePath.DeclinationTemplate);
+                    break;
+                default:
```

Comment:
better to write WaitingList, and in default throw InvalidArgumentException, with message Email is not sent when user is transfered in status {...} and pass status to {}